app/src/main/res/xml/settings.xml

```diff
@@ -29,9 +29,17 @@
     <SwitchPreference
         android:key="copyToClipboard"
         android:title="@string/copy_to_clipboard_setting_title"
-        android:summary="@string/copy_to_clipboard_setting_summary"
+        android:summary="Automatically add scanned information to your copy &amp; paste clipboard."
```

Comment:
+ 💯  thanks for spotting this. @nahojjjen could you please review your PRs to avoid this before declaring them as ready to merge?